lib/Controller/Stats.php

```diff
@@ -582,56 +583,70 @@ public function export()
         }
 
         // Get result set
-        $resultSet =  $this->timeSeriesStore->getStats([
-            'fromDt'=> $fromDt,
-            'toDt'=> $toDt,
+        $resultSet = $this->timeSeriesStore->getStats([
```

Comment:
If I am reading this correctly, we have to run the entire query in order to get the count?

We can certainly make that more elegant in MySQL by just running the `COUNT()` statement with filters - perhaps there is nothing we can do in Mongo though?